patches/tModLoader/Terraria.ModLoader.UI/Interface.cs

```diff
@@ -74,7 +74,7 @@ internal static class Interface
 				buttonNames[buttonIndex] = Language.GetTextValue("tModLoader.MenuModSources");
 				if (selectedMenu == buttonIndex) {
 					Main.PlaySound(10, -1, -1, 1);
-					Main.menuMode = ModCompile.DeveloperModeReady(out var _) ? modSourcesID : developerModeHelpID;
```

Comment:
What's exactly the issue with this? This is completely fine.

Reply:
Not really an issue. VS just underlines it and says it's unused